### PR TITLE
introduction + s/ServiceWorker/service worker

### PR DESCRIPTION
**Pull request by bsittler**

### CODE REVIEW

**Comment by bsittler** [comment]
Please take a look, @domenix @slightlyoff @inexorabletash @annevk - this adds an introduction to the explainer and is intended to succinctly clarify what this is and what motivates it. I'm especially interested in any additional background you might have on problems the existing synchronous APIs have and in whether this asynchronous API addresses them sufficiently to lighten the web platform maintenance burden should it be adopted

File: explainer.md [hunk 1]
```diff
@@ -1,6 +1,11 @@
...
+
+[HTTP cookies](https://tools.ietf.org/html/rfc6265) have, since their [origins at Netscape (preserved by archive.org)](https://web.archive.org/web/0/http://wp.netscape.com/newsref/std/cookie_spec.html), provided a [valuable state-management mechanism](http://www.montulli-blog.com/2013/05/the-reasoning-behind-web-cookies.html) for the web. Unfortunately the synchronous single-threaded script-level interface to them - also originating in the Netscape browser -  [has been a source of complexity and performance woes](https://lists.w3.org/Archives/Public/public-whatwg-archive/2009Sep/0083.html) exacerbated by the move in many browsers from a single-process, single-threaded model without a general expectation of responsiveness for scripted event handling while processing cookie operations to the modern web which strives for smoothly responsive high performance in a multithreaded, multi-process environment where a cookie operation in one part of a web application cannot block the rest of the application, web origin, or the browser as a whole.
```
Comment (inexorabletash) [inline explainer.md]: The part beginning with "Unfortunately" is a very long sentence. Maybe break it up.

**Comment by inexorabletash** [comment]
lgtm with just a couple of editorial nits

File: explainer.md (hunk 1 above)
Comment (bsittler) [inline explainer.md]: Better now? I wasn't sure how completely to overhaul it.

File: explainer.md [hunk 2]
```diff
@@ -471,9 +476,9 @@ Prefix rules are also enforced in write operations by this API, but may not be e
 
 ### URL scoping
 
-Although a ServiceWorker cannot directly access cookies today, it can already use controlled rendering of in-scope HTML and script resources to inject cookie-monitoring code under the remote control of the ServiceWorker, remotely controlled using postMessage, the caches API, or IndexedDB. This means that cookie access inside the scope of the ServiceWorker is technically possible already, it's just not very convenient.
+Although a service worker script cannot directly access cookies today, it can already use controlled rendering of in-scope HTML and script resources to inject cookie-monitoring code under the remote control of the service worker script, remotely controlled using postMessage, the caches API, or IndexedDB. This means that cookie access inside the scope of the service worker is technically possible already, it's just not very convenient.
```
Comment (bsittler) [inline explainer.md]: Done.

**Comment by bsittler** [comment]
I'm going ahead and merging this but feel free to file issues/send pull requests if you would like to see further charges

**Pull request merged**